### PR TITLE
Desktop design

### PR DESCRIPTION
**Pull request by nadja-po**
Added desktop design

### CODE REVIEW

File: src/components/atoms/Title/style.scss [hunk 1]
```diff
@@ -17,11 +17,12 @@
             font-size: 12px;
             letter-spacing: 1px;
             text-transform: uppercase;
-            line-height: 15px;
+            line-height: 25px;
 
             &--black {
                 @extend .title--subtitle--small;
                 color: var(--main-color);
+                margin: 10px 10px;
```
Comment (Adele-Locmele) [inline src/components/atoms/Title/style.scss:25]: Can be shortened to margin: 10px;

Comment (leleleadele) [inline src/components/atoms/Title/style.scss]: can be shortened to margin: 10px;

File: src/components/molecules/Footer/index.js [hunk 2]
```diff
@@ -9,17 +9,19 @@ function Footer({
...
-          <a href="#" key={item.id}>
-            <Icon
-              iconVariant="rounded"
-              src={iconsFooter && require(`../../../images/${iconsFooter[index].src}`).default}
-              alt={iconsFooter && iconsFooter[index].alt}
-            />
-          </a>
-        ))}
+      <div className="footer__icons">
+        <Title title={textFooter} titleVariant="subtitle--small--black" />
```
Comment (leleleadele) [inline src/components/molecules/Footer/index.js:13]: The naming here `subtitle--small--black` looks a bit unconventional IMO.

File: src/components/organisms/Menu/style.scss [hunk 3]
```diff
@@ -1,86 +1,145 @@
...
+        &__list {
+            padding: 0 10px 10px;
+            text-transform: uppercase;
+            font-size: 12px;
+            letter-spacing: 1px;
+            font-family: var(--main-text-font);
+            cursor: pointer;
+            justify-self: end;
         
             li {
```
Comment (leleleadele) [inline src/components/organisms/Menu/style.scss]: A safer approach is to style classes, not whole tag selectors.

**Review by artikvatruwka — APPROVED** [review]

File: src/components/organisms/Menu/style.scss (hunk 3 above)
Comment (nadja-po) [reply src/components/organisms/Menu/style.scss]: Changed.

File: src/components/molecules/Footer/index.js (hunk 2 above)
Comment (nadja-po) [reply src/components/molecules/Footer/index.js:13]: Why? There are 2 subtitle modificators: small and black.

File: src/components/atoms/Title/style.scss (hunk 1 above)
Comment (nadja-po) [reply src/components/atoms/Title/style.scss]: Fixed

**Pull request merged**